ui/src/testing/stubs/notebooks-cluster-api-stub.ts

```diff
@@ -0,0 +1,19 @@
+import {ClusterApi} from 'notebooks-generated/fetch';
+
+
+export class NotebooksClusterApiStub extends ClusterApi {
+
+  constructor() {
+    super(undefined, undefined, (..._: any[]) => {
+      throw Error('cannot fetch in tests');
+    });
+  }
+
+  public startCluster(googleProject: string, clusterName: string,
```

Comment:
I don't see startCluster() or setCookie() called anywhere. Are they in use indirectly somehow?

Reply:
They are both used by the NotebookRedirect component